dll/win32/browseui/lang/hu-HU.rc

```diff
@@ -0,0 +1,355 @@
+/*
+ * Copyright 2009 Andrew Hill
```

Comment:
I think it's not necessary to copy this license header from en-US translation file. You can add your own, as it was done here:
- https://git.reactos.org/?p=reactos.git;a=blob;f=dll/win32/browseui/lang/pl-PL.rc;hb=c082d08cf34cf09e003009dc5862fc7e0a6c38c1
- https://git.reactos.org/?p=reactos.git;a=blob;f=dll/win32/browseui/lang/es-ES.rc;hb=c082d08cf34cf09e003009dc5862fc7e0a6c38c1